src/MiniProfiler.Shared/ProfileProviders/DefaultProfilerProvider.cs

```diff
@@ -1,7 +1,8 @@
 ﻿#if NET45
 using System;
 using System.Runtime.Remoting.Messaging;
-using System.Web;
+using System.Web; 
+using StackExchange.Profiling.Helpers.Net45;
```

Comment:
Since there is AsyncLocal polyfill we can keep the implementation of the provider the same - it will be either using the polyfill or .net version of AsyncLocal